components/src/dynamo/common/tests/multimodal/test_lazy_imports.py

```
@@ -0,0 +1,142 @@
+import pytest
+
+pytestmark = [
+    pytest.mark.unit,
+    pytest.mark.gpu_0,
+    pytest.mark.pre_merge,
+]
+
+
+def _run_in_subprocess(script: str) -> None:
+    """Run ``script`` in a fresh interpreter, failing the test on nonzero exit."""
+    result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)